tests/vm_test.go

```diff
@@ -441,8 +441,8 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			}, 300*time.Second, 2*time.Second).Should(BeZero(), "The VirtualMachineInstance did not disappear")
 		},
 			table.Entry("with ContainerDisk", newVirtualMachineInstanceWithContainerDisk),
-			table.Entry("[Serial][rook-ceph]with OCS Filesystem Disk", newVirtualMachineInstanceWithOCSFileDisk),
-			table.Entry("[Serial][rook-ceph]with OCS Block Disk", newVirtualMachineInstanceWithOCSBlockDisk),
+			table.Entry("[Serial][block-storage]with Filesystem Disk", newVirtualMachineInstanceWithFileDisk),
```

Comment:
The `[block-storage]` conflicts with the `newVirtualMachineInstanceWithFileDisk` on the same line. I think you can remove the block storage filtering